addons/web/tooling/hooks/pre-commit

```diff
@@ -1,13 +1,16 @@
 #!/bin/bash
+current_git=$(git rev-parse --show-toplevel)
 tooling_dir=$(cd -- "$(dirname "$0")" &> /dev/null && cd .. && pwd)
 if ! cmp -s -- "$tooling_dir/_package.json" package.json; then
     echo "Your package.json is out of date, reloading the tooling using the reload script"
-    "$tooling_dir/reload.sh"
+    python3 "$tooling_dir"/enable.py nodejs remove --force-node --addons-path=$current_git
+    python3 "$tooling_dir"/enable.py nodejs install --force-node --addons-path=$current_git
 elif
     ! cmp -s -- "$tooling_dir/_eslintignore" .eslintignore ||
     ! cmp -s -- "$tooling_dir/_eslintrc.json" .eslintrc.json
 then
     echo "Some of your eslint/prettier config files are out of date, refreshing them using the refresh script"
-    "$tooling_dir/refresh.sh"
+    python3 "$tooling_dir"/enable.py nodejs remove --addons-path=$current_git
+    python3 "$tooling_dir"/enable.py nodejs install --addons-path=$current_git
```

Comment:
Seems like this does a full reinstall instead of just a refresh? Might want to merge this with the previous case then.

Reply:
the code just recopy eslintrc and some other, without retriggering npm install